ios/NebulaNetworkExtension/Site.swift

```diff
@@ -151,18 +219,17 @@ struct Site: Codable {
     var cipher: String
     var sortKey: Int
     var logVerbosity: String
-    var connected: Bool?
+    var connected: Bool? //TODO: active is a better name
```

Comment:
Does this actually need to be optionally null, or can this default false? I was surprised to see the `??` for this elsewhere in the code.

Reply:
Pedantically, we don't know the state of this until we query the manager for its state and there are ways to hydrate this class without having access to a manger.